kubectl-pg/cmd/addDb.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// addDbCmd represents the addDb command
+var addDbCmd = &cobra.Command{
+	Use:   "add-db",
+	Short: "Adds a DB and it's owner to a Postgres cluster. The owner role is created if it does not already exist",
+	Long:  `Adds a DB and it's owner to the cluster owner needs to be added with -o flag.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) > 0 {
+			dbName := args[0]
+			dbOwner, _ := cmd.Flags().GetString("owner")
+			clusterName, _ := cmd.Flags().GetString("clusterName")
+			addDb(dbName, dbOwner, clusterName)
+		} else {
+			fmt.Println("database name can't be empty.")
```

Comment:
what if someone does:

```golang
kubectl pg add-db   -o myowner -c mycluster"
```

this error message will not show but the command is still malformed

Reply:
![Screenshot from 2019-07-01 01-58-45](https://user-images.githubusercontent.com/25104868/60401780-fd1cac00-9ba3-11e9-960d-df89dc3e038b.png)

Error is caught here with the cmd you specified.